CookuPots/Screens/Categories/CategoriesVC.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  CategoriesVC.swift
+//  CookuPots
+//
+//  Created by Sebulla on 23/03/2022.
+//
+
+import UIKit
+
+class CategoriesVC: UICollectionViewController {
+    
+    let searchBar = UISearchBar()
+    
+    static let categoryHeaderId = "categoryHeaderId"
+    static let recomendedHeaderId = "recomendedHeaderId"
+    let headerId = "headerId"
+    let recommendedId = "recommendedId"
+    let CollectionCellId = "CollectionCellId"
+    
+    
+    init() {
+        super.init(collectionViewLayout: CategoriesVC.createLayout())
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureUI()
+        
+        collectionView.backgroundColor = .white
+        
+        navigationItem.title = "Cook U Pots"
+        
+        // how to make large title or customize title
```

Comment:
If you want to keep a comment for future, you can use
`// TODO: Fix large title` or
`// FIXME: Title is small at start`